### PR TITLE
[Feature] GDPR Command

### PR DESCRIPTION
**Pull request by Sanduhr32**
Dev only command for deleting user data after they request data removal.

- [x] Delete all giveaways hosted by them
- [ ] ~~Remove reactions from database~~ soon
- [ ] ~~Revoke their VIP Status?~~ user ids aren't PI

### CODE REVIEW

File: src/main/java/com/jagrosh/giveawaybot/Bot.java [hunk 1]
```diff
@@ -123,6 +133,23 @@ public boolean deleteGiveaway(long channelId, long messageId)
         catch(Exception ignore) {}
         return database.giveaways.deleteGiveaway(messageId);
     }
+
+    public RestJDA startRestJDA() {
```
Comment (ghost) [inline src/main/java/com/jagrosh/giveawaybot/Bot.java]: This sort of resembles a singleton pattern, should this be turned private or does it have other accesses outside the class?

**Review by ghost — APPROVED** [review]
LGTM

File: src/main/java/com/jagrosh/giveawaybot/Bot.java (hunk 1 above)
Comment (Sanduhr32) [reply src/main/java/com/jagrosh/giveawaybot/Bot.java]: True yes!

**Comment by Sanduhr32** [comment]
I did clean it up more and removed stupidity that occured due to minor focus on writing code in the most optimal way.

LGTM now as well.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**